src/store/reducers/queryResults.js

```diff
@@ -162,6 +174,40 @@ export function matchNewPostIntoQueryResults (state, { id, type, networkSlug, co
   }, state, communities)
 }
 
+export function matchNewTopicIntoQueryResults (state, { id, isDefault, networkSlug, communityTopics }) {
```

Comment:
This is a very exciting amendment. Good work! 🥇  ...and we may very well be right near the end of life on dear old `queryResults`